app/helpers/application_helper.rb

```diff
@@ -1,4 +1,14 @@
 module ApplicationHelper
+
+  # Used to generate links to the policies finder
+  def policies_filter_path(query_params = nil)
```

Comment:
Happy to rename it. `policies_finder_path`? Also happy to remove the empty `query_params` path as well.

I guess extracting the method is jumping a bit ahead given that it's only called in one template, but I think an extracted method gives us better encapsulation over a hardcoded path and query params in the view template.